backend/src/main/java/com/speakfluid/backend/usecases/TranscriptAnalysisInteractor.java

```diff
@@ -0,0 +1,92 @@
+package com.speakfluid.backend.usecases;
```

Comment:
Good idea, I was not a fan of naming a class UsecaseInteractor since it didn't make much sense to me